### PR TITLE
new icon: adonisjs (original, original-wordmark)

### PR DESCRIPTION
**Pull request by thetutlage**
Hello 👋

Submitting the icon for https://adonisjs.com

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+      "tags": [
+          "nodejs",
+          "framework"
+      ],
+      "versions": {
+          "svg": [
+              "original"
+          ],
+          "font": [
+              "plain"
```
Comment (Panquesito7) [inline devicon.json:13]: There's no plain version, so we should use the `original` version instead and add it as an alias.

```suggestion
              "original"
```

File: icons/adonisjs/adonisjs-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M0 64c0 51.617 12.383 64 64 64 51.617 0 64-12.383 64-64 0-51.617-12.383-64-64-64C12.383 0 0 12.383 0 64zm25.808 13.295l20.075-45.621C49.27 23.984 55.788 19.813 64 19.813c8.212 0 14.729 4.17 18.118 11.86l20.074 45.622c.912 2.216 1.694 5.084 1.694 7.56 0 11.34-7.951 19.291-19.292 19.291-3.862 0-6.93-.985-10.035-1.983-3.182-1.022-6.403-2.057-10.559-2.057-4.108 0-7.408 1.044-10.653 2.071-3.138.993-6.225 1.969-9.941 1.969-11.34 0-19.292-7.95-19.292-19.29 0-2.477.783-5.345 1.695-7.561zM64 39.625l-19.813 44.84c5.866-2.738 12.644-4.041 19.813-4.041 6.909 0 13.947 1.303 19.552 4.04L64 39.626z" fill="#5A45FF"/></svg>
```
Comment (Panquesito7) [inline icons/adonisjs/adonisjs-original.svg:1]:
```suggestion
<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M0 64c0 51.617 12.383 64 64 64 51.617 0 64-12.383 64-64 0-51.617-12.383-64-64-64C12.383 0 0 12.383 0 64zm25.808 13.295 20.075-45.621C49.27 23.984 55.788 19.813 64 19.813c8.212 0 14.729 4.17 18.118 11.86l20.074 45.622c.912 2.216 1.694 5.084 1.694 7.56 0 11.34-7.951 19.291-19.292 19.291-3.862 0-6.93-.985-10.035-1.983-3.182-1.022-6.403-2.057-10.559-2.057-4.108 0-7.408 1.044-10.653 2.071-3.138.993-6.225 1.969-9.941 1.969-11.34 0-19.292-7.95-19.292-19.29 0-2.477.783-5.345 1.695-7.561zM64 39.625l-19.813 44.84c5.866-2.738 12.644-4.041 19.813-4.041 6.909 0 13.947 1.303 19.552 4.04L64 39.626z" fill="#5A45FF"/></svg>

```

File: devicon.json [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
+          ]
+      },
+      "color": "#5A45FF",
+      "aliases": []
```
Comment (Panquesito7) [inline devicon.json:17]:
```suggestion
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Good work! The icon looks good. 👍
Please add a `wordmark` version as shown in [their website](https://adonisjs.com/).

image

**Comment by thetutlage** [comment]
> Please add a wordmark version as shown in their website.

Sorry, I am not much into SVGs. But can you guide how to fit that wordmark logo into a "128 x 128" viewbox?

File: devicon.json (hunk 1 above)
Comment (thetutlage) [reply devicon.json]: Sorry, overlooked it :)

File: icons/adonisjs/adonisjs-original.svg (hunk 2 above)
Comment (thetutlage) [reply icons/adonisjs/adonisjs-original.svg]: Fixed

File: devicon.json (hunk 3 above)
Comment (thetutlage) [reply devicon.json]: Added :)

File: icons/adonisjs/adonisjs-original-wordmark.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...823 1.912 0 3.18 1.495 3.18 3.823s-1.268 3.823-3.18 3.823c-1.892 0-3.16-1.495-3.16-3.823zm41.713 3.009c0 .7-.53 1.117-1.874 1.117-1.514 0-2.46-.795-2.46-1.76h-3.785c.094 2.611 2.574 4.56 6.17 4.56 3.406 0 5.583-1.513 5.583-4.2 0-2.007-1.287-3.408-3.842-3.919l-2.744-.549c-.833-.17-1.325-.586-1.325-1.135 0-.814.833-1.25 1.892-1.25 1.514 0 2.101.701 2.101 1.534h3.596c-.095-2.46-2.025-4.334-5.621-4.334-3.35 0-5.621 1.684-5.621 4.24 0 2.308 1.495 3.35 3.729 3.822l2.725.587c.889.19 1.476.568 1.476 1.287zm-14.062-9.633h3.785v13.248h-3.785V59.888zm-11.583 0v1.381c.965-1.116 2.252-1.684 3.842-1.684 2.972 0 4.902 1.93 4.902 4.92v8.63h-3.785V65.3c0-1.608-.909-2.573-2.479-2.573-1.571 0-2.48.965-2.48 2.574v7.835h-3.785V59.888h3.785zm-12.907-.303c-4.032 0-7.022 2.914-7.022 6.927 0 4.012 2.99 6.927 7.022 6.927 4.05 0 7.04-2.915 7.04-6.927 0-4.013-2.99-6.927-7.04-6.927zm-31.57 3.085c-1.93 0-3.198 1.533-3.198 3.88 0 2.29 1.268 3.785 3.199 3.785 1.95 0 3.198-1.495 3.198-3.804 0-2.347-1.249-3.861-3.198-3.861zm-7.059 3.88c0 4.202 2.631 6.889 6.246 6.889 1.722 0 2.952-.55 3.69-1.647v1.344h3.786V59.888h-3.785v1.381c-.757-1.116-2.007-1.684-3.691-1.684-3.615 0-6.246 2.687-6.246 6.965z" fill="#000"/></svg>
```
Comment (Panquesito7) [inline icons/adonisjs/adonisjs-original-wordmark.svg:1]: This suggestion optimizes the SVG and makes its size smaller.
```suggestion
<svg viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M0 63.626c0 15.828 3.797 19.626 19.626 19.626 15.828 0 19.626-3.798 19.626-19.626C39.252 47.797 35.454 44 19.626 44 3.797 44 0 47.797 0 63.626zm7.914 4.077 6.156-13.99c1.04-2.358 3.038-3.637 5.556-3.637 2.518 0 4.516 1.279 5.556 3.637l6.155 13.99c.28.68.52 1.559.52 2.318 0 3.478-2.438 5.916-5.916 5.916-1.184 0-2.125-.302-3.077-.608-.976-.314-1.964-.631-3.238-.631-1.26 0-2.272.32-3.267.635-.962.304-1.909.604-3.049.604-3.477 0-5.915-2.438-5.915-5.916 0-.76.24-1.639.52-2.318zM19.626 56.15 13.55 69.9c1.799-.84 3.877-1.239 6.076-1.239 2.118 0 4.277.4 5.995 1.24l-5.995-13.75zm92.552-2.547c-1.363 0-2.347.965-2.347 2.31 0 1.343.984 2.308 2.347 2.308 1.362 0 2.347-.965 2.347-2.309 0-1.344-.985-2.309-2.347-2.309zM69.253 73.44c-3.634 0-6.264-2.687-6.264-6.89 0-4.277 2.63-6.964 6.264-6.964 1.685 0 2.934.568 3.672 1.684v-7.06h3.785v18.927h-3.785v-1.344c-.72 1.098-1.968 1.647-3.672 1.647zm-2.403-6.89c0-2.346 1.268-3.88 3.198-3.88 1.95 0 3.199 1.515 3.199 3.862 0 2.309-1.25 3.804-3.199 3.804-1.93 0-3.198-1.495-3.198-3.785zm15.803-.037c0-2.328 1.268-3.823 3.16-3.823 1.912 0 3.18 1.495 3.18 3.823s-1.268 3.823-3.18 3.823c-1.892 0-3.16-1.495-3.16-3.823zm41.713 3.009c0 .7-.53 1.117-1.874 1.117-1.514 0-2.46-.795-2.46-1.76h-3.785c.094 2.611 2.574 4.56 6.17 4.56 3.406 0 5.583-1.513 5.583-4.2 0-2.007-1.287-3.408-3.842-3.919l-2.744-.549c-.833-.17-1.325-.586-1.325-1.135 0-.814.833-1.25 1.892-1.25 1.514 0 2.101.701 2.101 1.534h3.596c-.095-2.46-2.025-4.334-5.621-4.334-3.35 0-5.621 1.684-5.621 4.24 0 2.308 1.495 3.35 3.729 3.822l2.725.587c.889.19 1.476.568 1.476 1.287zm-14.062-9.633h3.785v13.248h-3.785V59.888zm-11.583 0v1.381c.965-1.116 2.252-1.684 3.842-1.684 2.972 0 4.902 1.93 4.902 4.92v8.63h-3.785V65.3c0-1.608-.909-2.573-2.479-2.573-1.571 0-2.48.965-2.48 2.574v7.835h-3.785V59.888h3.785zm-12.907-.303c-4.032 0-7.022 2.914-7.022 6.927 0 4.012 2.99 6.927 7.022 6.927 4.05 0 7.04-2.915 7.04-6.927 0-4.013-2.99-6.927-7.04-6.927zm-31.57 3.085c-1.93 0-3.198 1.533-3.198 3.88 0 2.29 1.268 3.785 3.199 3.785 1.95 0 3.198-1.495 3.198-3.804 0-2.347-1.249-3.861-3.198-3.861zm-7.059 3.88c0 4.202 2.631 6.889 6.246 6.889 1.722 0 2.952-.55 3.69-1.647v1.344h3.786V59.888h-3.785v1.381c-.757-1.116-2.007-1.684-3.691-1.684-3.615 0-6.246 2.687-6.246 6.965z" fill="#000"/></svg>

```

Comment (thetutlage) [reply icons/adonisjs/adonisjs-original-wordmark.svg]: Done. Thanks :)

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thank you for your contribution! 😄👍🎉

**Comment by thetutlage** [comment]
Thanks for guiding and accepting the PR :)

**Pull request merged**